### PR TITLE
Separate linkers for different scripts

### PR DESCRIPTION
**Pull request by nazrhom**

### CODE REVIEW

**Review by the-headless-ghost — APPROVED** [review]

File: ytxp-plutarch/src/Cardano/YTxP.hs [hunk 1]
```diff
@@ -38,49 +40,77 @@ data YTxPParams = YTxPParams
...
+validatorLinker :: Linker YTxPParams (ScriptExport YTxPParams)
+validatorLinker = do
   info <- getParam
 
   yieldingValidator <-
     fetchTS
       @'ThreeArgumentScript
       @'[CurrencySymbol]
-      "djed:yieldingV"
+      "ytxp:yieldingV"
```
Comment (albertodvp) [inline ytxp-plutarch/src/Cardano/YTxP.hs:51]: I personally find it a bit strange that sometimes we abbreviate `validator` with `v` and some other times it's the full. Also I think it's not very consistent:
- `validatorLinker`
- `mpLinker`



I'd probably prefer either:
- `vLinker`
- `mpLinker`
or
- `validatorLinker`
- `mintingPolicyLinke`

But I understand it's a stylistic thing and I'm approving this anyway, feel free to update the PR or ignore this :relaxed:

Comment (albertodvp) [inline ytxp-plutarch/src/Cardano/YTxP.hs:51]: I personally find it a bit strange that sometimes we abbreviate `validator` with `v` and some other times it's the full. Also I think it's not very consistent:
- `validatorLinker`
- `mpLinker`



I'd probably prefer either:
- `vLinker`
- `mpLinker`
or
- `validatorLinker`
- `mintingPolicyLinke`

But I understand it's a stylistic thing and I'm approving this anyway, feel free to update the PR or ignore this :relaxed:

**Review by albertodvp — APPROVED** [review]
I don't know much about ply so Edi is probably better if you need a precise review but if you want to iterate fast and feel like merging it that works for me

File: ytxp-plutarch/src/Cardano/YTxP.hs (hunk 1 above)
Comment (nazrhom) [reply ytxp-plutarch/src/Cardano/YTxP.hs:51]: you are right. I used the convention of abbreviating mp and sv but not validator (which is a bit weird), then I noticed we had updated this to yieldingV so I stuck with that

**Pull request merged**